pkg/urbit/vere/io/ames.c

```diff
@@ -162,110 +171,102 @@ _ames_alloc(uv_handle_t* had_u,
 static void
 _ames_pact_free(u3_pact* pac_u)
 {
-  if ( 0 == pac_u->typ_y ) {          // ames packet
-  } else if ( 1 == pac_u->typ_y ) {  // fine request
-    c3_free(pac_u->req_u.pat_c);
-  } else {                         // fine response
-    c3_free(pac_u->res_u.pat_c);
-    c3_free(pac_u->res_u.dat_y);
+  switch ( pac_u->typ_y ) {
+    case PACT_AMES:
+      break;
+
+    case PACT_WAIL:
+      c3_free(pac_u->wal_u.ken_u.pat_c);
+      break;
```

Comment:
I believe we generally include braces for all cases, like so:

```suggestion
    case PACT_WAIL: {
      c3_free(pac_u->wal_u.ken_u.pat_c);
    } break;
```

Reply:
Roger that